product_docs/docs/pgd/5/quickstart/quick_start_docker.mdx

```diff
@@ -154,7 +154,7 @@ tpaexec configure democluster \
 ```
 
 You specify the PGD-Always-ON architecture (`--architecture PGD-Always-ON`), which
-sets up the configuration for [PGD 5's Always-on 
+sets up the configuration for [PGD's Always-on 
```

Comment:
We don't usually refer to the version number. Let me know if there was some reason it was used here.